docs/core/compatibility/core-libraries/6.0/xmlresolver-nullable.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Breaking change: XmlDocument.XmlResolver nullability change"
+description: Learn about the .NET 6 breaking change where the setter for XmlDocument.XmlResolver was made nullable.
+ms.date: 11/04/2021
+---
+# XmlDocument.XmlResolver nullability change
+
+The nullability for <xref:System.Xml.XmlDocument.XmlResolver?displayProperty=nameWithType> has changed in .NET 6. The property setter now accepts a nullable reference.
+
+## Previous behavior
+
+The setter for <xref:System.Xml.XmlDocument.XmlResolver?displayProperty=nameWithType> was annotated as not accepting `null`:
+
+```csharp
+public virtual System.Xml.XmlResolver XmlResolver { set { } }
+```
+
+## New behavior
+
+The setter for <xref:System.Xml.XmlDocument.XmlResolver?displayProperty=nameWithType> is annotated as accepting `null`:
+
+```csharp
+public virtual System.Xml.XmlResolver? XmlResolver { set { } }
+```
+
+## Version introduced
+
+6.0 RC 1
+
+## Type of breaking change
+
+This change can affect [binary compatibility](../../categories.md#binary-compatibility) if you've overridden this property in a derived type.
```

Comment:
Not sure about source vs. binary incompatibility for the derived type case.

Reply:
Adding @MadsTorgersen to be sure, but I think this would be only a source incompatibility change.

Reply:
I think it's more valuable to have this article out with .NET 6, so we can come back and tweak the wording on this if need be. I'm going to :shipit: